apps/OboeTester/app/src/main/java/com/mobileer/oboetester/AudioStreamTester.java

```diff
@@ -28,6 +28,8 @@ AudioStreamBase getCurrentAudioStream() {
     }
 
     public void open() throws IOException {
+        requestedConfiguration.setPackageName(BuildConfig.APPLICATION_ID);
+        requestedConfiguration.setAttributionTag("AudioTag"); // Match attribution in AndroidManifest.xml
```

Comment:
Do you need to read the value of "AudioTag" from the Manifest?

Reply:
+1 for reading the value from manifest.

Reply:
There can be multiple attribution tags in the manifest and there's no way to query it. Thus, we just need to type it twice

Reply:
There is API... https://developer.android.com/reference/android/content/pm/Attribution#getTag()